contracts/optimistic-ethereum/libraries/utils/Lib_RingBuffer.sol

```diff
@@ -1,9 +1,5 @@
 pragma solidity ^0.7.0;
 
-interface iRingBufferOverwriter {
```

Comment:
Overwriting is now push-based instead of pull-based, meaning some "owner" address will mark an index as ok to overwrite.